site/docs/latest/getting-started/installation.md

```diff
@@ -60,7 +66,9 @@ Command | Action
 `mvn verify` | Performs a wide variety of verification and validation tasks
 `mvn apache-rat:check` | Run Maven using the [Apache Rat](http://creadur.apache.org/rat/apache-rat-plugin/) plugin
 `mvn compile javadoc:aggregate` | Build Javadocs locally
-`mvn package assembly:single` | Build a complete distribution using the Maven [Assembly](http://maven.apache.org/plugins/maven-assembly-plugin/) plugin
+`mvn -am -pl bookkeeper-dist/server package` | Build a server distribution using the Maven [Assembly](http://maven.apache.org/plugins/maven-assembly-plugin/) plugin
```

Comment:
I don't understand why this changed? How is this better than straight "package assembly:single"?

Reply:
This is not related strictly to this issue, but currently the command is that one
we can fix it on master as follow up

Reply:
Ah, i see. The previous command doesn't actually work.